### PR TITLE
Set up page

### PR DESCRIPTION
**Pull request by idalmys**
Added footer and link pages

### CODE REVIEW

File: menu.html [hunk 1]
```diff
@@ -53,84 +51,56 @@ <h1>Cooks!!!</h1>
...
+    <div class="page-footer #b71c1c red darken-4">
+      © 2020 Copyright
     </div>
   </footer>
   <!--JavaScript at end of body for optimized loading-->
   <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
-  <!-- Firebase References -->
-  <script src="https://www.gstatic.com/firebasejs/6.0.4/firebase-app.js"></script>
-  <script src="https://www.gstatic.com/firebasejs/6.0.4/firebase-database.js"></script>
-  <script type="text/javascript" src="assets/javascript/home.js"></script>
```
Comment (dennistonc) [inline menu.html]: not sure if needed for the JS

**Review by dennistonc — CHANGES_REQUESTED** [review]
looks good, but check with monte if firebase is needed on that page

**Review by AshfordMonte — APPROVED** [review]
Looks good! I fixed the JS scripts for firebase and it's all working now

**Review by dennistonc — APPROVED** [review]
looks good

**Pull request merged**